src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/on--event--.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'on(event)'
+excerpt: 'xk6-browser: Browser.on method'
+---
+
+Detects events from the browser application.
+
+| Parameter | Type   | Description                                        |
+| --------- | ------ | -------------------------------------------------- |
+| event     | string | The only accepted event value is `'disconnected'`. |
+
+### Returns
+
+The returned promise will be resolved when the Browser disconnects from the browser application. Possible reasons for this might be happening as follows:
```

Comment:
```suggestion
The returned promise will be resolved when the [Browser](/javascript-api/xk6-browser/browser/) disconnects from the browser application. Possible reasons for this might be happening as follows:
```